docs/modifying_a_plugin.md

```diff
@@ -0,0 +1,28 @@
+# Modifying a plugin
+
+## Tutorial
+
+This is how you can modify an existing ILLIXR plugin
+
+1. Clone the repository for the component you want to modify. e.g: `git clone https://github.com/ILLIXR/audio_pipeline.git`
+
+2.  Modify the config file like this:
```

Comment:
Step 2 should be "make changes..." and then step 3 should be "modify config file".

For the sake of completeness and clarity, maybe step 4 to say "now when you run ILLIXR, the local audio component gets built and run".

And maybe there should be a step 5 as well describing what to do when you push your changes back up.

Reply:
Are step 2 and step 4 necessary? Its sort of implied.